test-requirements.txt

```
@@ -0,0 +1 @@
+git+https://github.com/OCA/contract@refs/pull/1312/head#subdirectory=setup/contract
```

```suggestion
odoo-addon-contract @ git+https://github.com/OCA/contract@refs/pull/1312/head#subdirectory=contract
```

Can you please place this in a separate commit? ITest requirements are not to be merged.